.github/actions/paths-filter/filters.yml

```diff
@@ -1,18 +1,41 @@
-rust:
+shared: &shared
   - .github/**
+
+rust:
+  - *shared
   - oxidation/**/*.rs
   - oxidation/**/*.toml
   - Cargo.toml
   - Cargo.lock
+
 python:
-  - .github/**
+  - *shared
   - parsec/**
   - tests/**
   - pyproject.toml
   - poetry.lock
   - build.py
+
 rust-ext:
+  - *shared
   - .github/**
   - src/**/*.rs
   - Cargo.toml
   - Cargo.lock
+
+ionic-client: &ionic-client
+  - oxidation/client/src/**
```

Comment:
Why not doing `oxidation/client/**` instead ?

Reply:
This would include every client like android, electron, ios and web

Reply:
Can we use a negative match instead ?
My fear is that this config file is well hidden so there is zero chance it will be modified when new files/folders are added, hence those new files should fall into the "common has been modified" match by default

Reply:
Sadly, `paths-filter` don't support negative match

Reply:
😭